examples/cantilever.ipynb

```diff
@@ -281,8 +215,8 @@
     "                δu = shape_value(cellvalues, q_point, i)\n",
     "                fe[i] += (δu ⋅ b) * dΩ\n",
     "                ɛC = ɛ[i] ⊡ C\n",
-    "                for j in 1:n_basefuncs\n",
-    "                    Ke[i, j] += (ɛC ⊡ ɛ[j]) * dΩ\n",
+    "                for j in i:n_basefuncs # assemble only upper half\n",
+    "                    Ke.data[i, j] += (ɛC ⊡ ɛ[j]) * dΩ # faster to assemble into the parent matrix\n",
```

Comment:
I do wrap it once now. The only thing with `.data` here is to avoid the `uplo` check in the `Symmetric` `setindex!` so I guess that’s no cost